engines/scalac/src/main/scala/scala/macros/internal/engines/scalac/trees/Abstracts.scala

```diff
@@ -615,9 +618,9 @@ trait Abstracts extends scala.macros.trees.Abstracts with Positions { self: Univ
           tparams: List[Type.Param],
           ctor: Ctor.Primary,
           templ: Template): Defn.Class = ???
-      def unapply(
-          gtree: Any): Option[(List[Mod], Type.Name, List[Type.Param], Ctor.Primary, Template)] =
-        ???
+      def unapply(gtree: Any): Option[(List[Mod], Type.Name, List[Type.Param], Ctor.Primary, Template)] = {
+       ???
+      }
```

Comment:
Files in this project are formatted with scalafmt, and that's how it decided to format this unapply :)